recipe/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+if [ `uname` == "Darwin" ]
+then
+    export LDFLAGS="-headerpad_max_install_names"
+fi
+
+./configure --prefix="${PREFIX}" \
+            --with-gmp="${PREFIX}"
```

Comment:
I think you are right. Was just having a rough time getting it to find the libraries.